android/KMEA/app/src/main/java/com/tavultesoft/kmea/cloud/impl/CloudKeyboardMetaDataDownloadCallback.java

```diff
@@ -266,11 +260,11 @@ private void handleKeyboardMetaData(MetaDataResult theKbData)
       }
       if (oskFontUrls != null) {
         for (String url : oskFontUrls) {
-          if (!urls.contains(url))
+          if (fontUrls==null || ! fontUrls.contains(url))
```

Comment:
Just checking if you had a test scenario that needed this change?

Reply:
But since you're already modified the line, can you add this chore?
```
if () {
}
```

That'll help future maintenance.

Reply:
One of the inspection tasks said, that it might be a bug because I changed the datatype of urls. So I changed the contains check back to a the list of strings.
Did not test it, but the check was broken after my last change